src/main/resources/static/api.yaml

```diff
@@ -2172,3 +2242,33 @@ components:
       properties:
         refreshToken:
           type: string
+    OtpVerificationRequest:
+      type: object
+      properties:
+        username:
+          type: string
+    OtpVerificationResponse:
+      type: object
+      properties:
+        sessionId:
+          type: string
```

Comment:
Need to send some info about when it's going to expire, to where we send the OTP (mobile/email) with respective value being masked.

Reply:
Where do we get that info from?

Reply:
Wherever you are going to send OTP, right now it's phone number and OTP expiry is 5 minutes configured in the OTP service.

Reply:
Done.